react/features/base/conference/middleware.js

```diff
@@ -589,7 +590,7 @@ function _syncReceiveVideoQuality({ getState }, next, action) {
 function _trackAddedOrRemoved(store, next, action) {
     const track = action.track;
 
-    if (track && track.local) {
+    if (track && track.local && track.mediaType !== MEDIA_TYPE.PRESENTER) {
```

Comment:
Can you add a TODO note here? In the future all track swapping should probably happen here instead of conference.js, but it's now nearly impossible to do, so let's leave a ccomment as a reminder.

Reply:
Sure